stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -61,6 +61,9 @@ private val LocalAttachmentFactories = compositionLocalOf<List<AttachmentFactory
 private val LocalAttachmentPreviewHandlers = compositionLocalOf<List<AttachmentPreviewHandler>> {
     error("No attachment preview handlers provided! Make sure to wrap all usages of Stream components in a ChatTheme.")
 }
+private val LocalQuoteAttachmentFactories = compositionLocalOf<List<AttachmentFactory>> {
```

Comment:
Should this maybe be `LocalQuotedAttachmentFactories` instead of `LocalQuoteAttachmentFactories`?